app/src/test/java/org/oppia/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -226,6 +239,127 @@ class StateFragmentLocalTest {
     }
   }
 
+  @Test
+  fun testStateFragment_submitTwoWrongAnswers_checkPreviousHeaderVisible() {
+    launchForExploration(FRACTIONS_EXPLORATION_ID_1).use {
+      startPlayingExploration()
+      playThroughState1()
+
+      submitTwoWrongAnswers()
+
+      onView(withId(R.id.previous_response_header)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testQuestionPlayer_submitTwoWrongAnswers_checkPreviousHeaderVisible() {
```

Comment:
The tests are good but we should implement then in a separate file something like `QuestionFragmentLocalTest`
Because this file is specifically for StateFragment which also tests audio player which is not a part of `QuestionFragmentLocalTest`

Reply:
Okay, Thanks :)